### ISSUE TITLE
Batch process  for adding new GTFS feeds that require authentication

### ISSUE THREAD

**Issue by emmambd** [MobilityData/mobility-feed-api#17]
## **User Story**

As an API consumer, I need to access new feeds that require API access without setting up my own authentication, so it's possible to discover more feeds at once. 

## **Acceptance Criteria**

- A batch process that automatically stores new GTFS feeds from the Mobility Database Github repo that have an `authentication_type` of `1` or `2`.
- Batch process should include all the steps for authenticated feeds outlined in [this flowchart](https://miro.com/app/board/uXjVM86LB1s=/?moveToWidget=3458764558857335430&cot=14)
![Public  Mobility Database API_ User Story Mapping - Flowchart for adding new feeds (1)](https://github.com/MobilityData/mobility-feed-api/assets/90632321/e4c06d60-9a44-48ea-91bf-a26ae8c965c3)

## **Materials and References**

- You can see the API authentication for different producer URL's by looking at the `authentication_info_url` in https://bit.ly/catalogs-csv
- [PR for when we added authenticated GTFS feeds to the Mobility Database](https://github.com/MobilityData/mobility-database-catalogs/pull/188)
- [PR that added no auth GTFS feed workflows to the Mobility Database catalogs repo](https://github.com/MobilityData/mobility-database-catalogs/pull/61)

**Comment by davidgamez** [comment]
Depends on #16

**Comment by emmambd** [comment]
Duplicate of https://github.com/MobilityData/mobility-feed-api/issues/16
